examples/worker/worker_util.py

```diff
@@ -1,44 +1,13 @@
 from conductor.client.automator.task_handler import TaskHandler
-from conductor.client.worker.worker import Worker
-from conductor.client.worker.worker_interface import WorkerInterface
-
 from examples.api import api_util
-from examples.worker import workers
 
 
 def start_workers() -> TaskHandler:
     task_handler = TaskHandler(
-        workers=[
-            create_worker_get_user_info(),
-            create_worker_send_email(),
-            create_worker_send_sms()
-        ],
-        configuration=api_util.get_configuration()
+        workers=[],
```

Comment:
Shouldn't we default workers to [] if not passed at all rather than asking the dev to pass an empty array?

Reply:
Should be solved by: https://github.com/conductor-sdk/conductor-python/pull/190